config/triplea/schemas/lobby_server.json

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Lobby server set",
+  "description": "A set of lobby servers",
+  "type": "array",
+  "items": {
+    "title": "Lobby server",
+    "description": "A lobby server",
+    "type": "object",
+    "properties": {
+      "version": {
+        "description": "The lobby server engine version",
+        "type": "string",
+        "pattern": "^\\d+\\.\\d+\\.\\d+\\.\\d+$"
+      },
+      "host": {
+        "description": "The lobby server host",
+        "type": "string",
+        "anyOf": [
+          { "format": "host-name" },
+          { "format": "ipv4" },
+          { "format": "ipv6" }
+        ]
+      },
+      "port": {
+        "description": "The TCP port on which the lobby server accepts connections",
+        "type": "number",
+        "minimum": 0,
+        "maximum": 65535
+      },
+      "message": {
```

Comment:
IIRC blank message == okay

The message is set on older servers to notify users to upgrade, or if the primary is taken down. The game may also use it as a trigger for a nicer, "the lobby is down" message.